ethcore/types/src/transaction/transaction.rs

```diff
@@ -441,20 +435,15 @@ impl SignedTransaction {
 	/// Try to verify transaction and recover sender.
 	pub fn new(transaction: UnverifiedTransaction) -> Result<Self, parity_crypto::publickey::Error> {
 		if transaction.is_unsigned() {
-			Ok(SignedTransaction {
-				transaction: transaction,
-				sender: UNSIGNED_SENDER,
```

Comment:
Can we remove this `const` now?

Reply:
What do you by `const`, remove this check? It's a fast path for the signature check.

Reply:
I meant to ask if this was the only place `UNSIGNED_SENDER` is used, but I checked myself and it's used all over so, no, we can't remove it.